### PR TITLE
initial code for model1

### PR DESCRIPTION
**Pull request by mudit1990**
DO NOT MERGE

### CODE REVIEW

File: code/model1.py [hunk 1]
```diff
@@ -1,9 +1,76 @@
...
 
 
-class Model1(Model):
+class Model1(nn.Module, Model):
+
+    def __init__(self, epochs=10):
+        self.epochs = epochs
+        super(Model1, self).__init__()
+        # embedding lookup for 150 users each have 100 dimension representation
+        self.embedding_layer = nn.Embedding(150, 100)
```
Comment (mohit-surana) [inline code/model1.py]: Let's reduce this to something much smaller like 20 or 50

File: code/model1.py [hunk 2]
```diff
@@ -1,9 +1,76 @@
...
+        # first hidden layer, linear layer with weights 200x500
+        # this should be the size of <sender+receiver representation>
+        self.h1_layer = nn.Linear(200, 500)
```
Comment (mohit-surana) [inline code/model1.py]: Similarly, here as well

File: code/utils.py [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+
+
+def get_userid(u):
+    """
+    gets the unique user_id for every user. Returns none if the mapping is not present
+    :param u:
+    :return:
+    """
+    if u in user_id_lookup:
+        return user_id_lookup[u]
```
Comment (mohit-surana) [inline code/utils.py:17]: I'm guessing this is yet to be implemented.
Can we not use email ids directly to indeed the embeddings?

**Review by mohit-surana — COMMENTED** [review]
Seems promising!

File: code/model1.py (hunk 1 above)
Comment (mudit1990) [reply code/model1.py]: yeah ... it was just an approx number

File: code/utils.py (hunk 3 above)
Comment (mudit1990) [reply code/utils.py:17]: nope ... nn.Embeddings expects a long tensor, it can't deal with strings

**Pull request closed without merging**